hyperopt/fmin.py

```diff
@@ -97,6 +98,17 @@ def partial(fn, **kwargs):
     return rval
 
 
+def __objective_fmin_wrapper(func):
+    """
+    Wrap the objective function on a dict to kargs
```

Comment:
the typical way to abbreviate this is args and kwargs, but not kargs

Reply:
Indeed... I'll update the code